folder_paths.py

```diff
@@ -243,6 +243,9 @@ def map_filename(filename):
     def compute_vars(input, image_width, image_height):
         input = input.replace("%width%", str(image_width))
         input = input.replace("%height%", str(image_height))
+        input = input.replace("%year%", time.strftime("%Y"))
+        input = input.replace("%month%", time.strftime("%m"))
+        input = input.replace("%day%", time.strftime("%d"))
```

Comment:
very weird edge case, but I believe it's possible for time to change between these 3 lines - if you run at the exact stroke of midnight you could get a weird value

Reply:
great callout